download_and_install/compiling_from_source.md

```diff
@@ -0,0 +1,242 @@
+Compiling From Source
+=====================
+
+OLA source code lives on [GitHub](https://github.com/OpenLightingProject/ola).
+Stable releases are available from the [releases page](https://github.com/OpenLightingProject/ola/releases);
+for bleeding-edge development releases, run `git clone https://github.com/OpenLightingProject/ola.git`.
```

Comment:
We probably want to mention branches here too, depending on what they're doing they may want master or a stable parent branch (e.g. 0.10 currently).